node_modules/oae-zencoder/lib/api.js

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var util = require('util');
+
+var log = require('oae-logger').logger('oae-zencoder');
+var PreviewContext = require('oae-preview-processor/lib/model').PreviewContext;
+var PreviewProcessorAPI = require('oae-preview-processor');
+var PreviewUtil = require('oae-preview-processor/lib/util');
+
+var ZencoderDAO = require('./internal/dao');
+
+var config = null;
+
+/**
+ * Set the default config. This can be overridden for individual tenants
+ *
+ * @return {Object} Zencoder configuration
+ */
+var init = module.exports.init = function(_config) {
+    config = _config;
+};
+
+/**
+ * @return {Object}    The configuration object that is currently in use.
+ */
+var getConfiguration = module.exports.getConfiguration = function() {
+    return config;
+};
+
+/**
+ * Save data for the completed preview when encoding has been completed.
+ *
+ * @param  {Object}     notification    The notification data received from Zencoder
+ * @param  {Function}   callback        Standard callback function
+ * @param  {Object}     callback.err    An error that occurred, if any
+ */
+
+var encodingCompleted = module.exports.encodingCompleted = function(notification, callback) {
+    log().info('Received encoding completed notification from Zencoder');
```

Comment:
Can we wait until we know for what content item or what tenant we have received a notification? This log message on its own seems only useful when doing development and not for debugging a production system when there are are potentially many encoding complete messages coming through.